
package com.laegler.openbanking.soap.service;

import javax.xml.ws.WebFault;


/**
 * Unauthorized
 *
 * This class was generated by Apache CXF 3.3.3
 * 2019-10-21T07:01:27.394+13:00
 * Generated source version: 3.3.3
 */

@WebFault(name = "PUT_Updatethebusinessorthetechnicalcontactsforthegivenorganisation_401", targetNamespace = "http://laegler.com/openbanking/soap/model")
public class PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation401 extends Exception {

    private java.lang.Object putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation401;

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation401() {
        super();
    }

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation401(String message) {
        super(message);
    }

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation401(String message, java.lang.Throwable cause) {
        super(message, cause);
    }

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation401(String message, java.lang.Object putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation401) {
        super(message);
        this.putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation401 = putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation401;
    }

    public PUTUpdatethebusinessorthetechnicalcontactsforthegivenorganisation401(String message, java.lang.Object putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation401, java.lang.Throwable cause) {
        super(message, cause);
        this.putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation401 = putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation401;
    }

    public java.lang.Object getFaultInfo() {
        return this.putUpdatethebusinessorthetechnicalcontactsforthegivenorganisation401;
    }
}
